packages/lesmis/src/components/MainMenu.js

```diff
@@ -0,0 +1,141 @@
+import { LightIconButton } from '@tupaia/ui-components';
+import { FlexEnd } from './Layout';
+
+const StyledList = styled(List)`
+  width: 22.5rem;
+
+  .MuiListItem-gutters {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  .MuiListItemIcon-root {
+    min-width: 2.5rem;
+  }
+
+  .MuiListItemText-primary {
+    font-size: 1.125rem;
+    line-height: 140%;
+  }
+`;
+
+const MenuSection = styled.div`
+  display: flex;
+  align-items: center;
+  height: 100%;
+  margin-right: 1.2rem;
+  padding-right: 0.3rem;
+  margin-left: -1.3rem;
+  border-right: 1px solid rgba(0, 0, 0, 0.2);
+
+  .MuiIconButton-root {
+    padding: 1rem;
+  }
+`;
+
+const MenuTray = styled(FlexEnd)`
+  padding: 0.375rem 0.375rem 0 0;
+  margin-bottom: -0.375rem;
+`;
+
+const MenuHeading = styled(Typography)`
+  font-weight: normal;
+  font-size: 2rem;
+  line-height: 140%;
+  margin-bottom: 0.5rem;
+`;
+
+const StyledDivider = styled(Divider)`
+  margin: 0.375rem 1.875rem;
+`;
+
+const TupaiaText = styled(Typography)`
+  color: ${props => props.theme.palette.text.secondary};
+  margin-top: 0.5rem;
+  font-size: 0.75rem;
+  line-height: 140%;
+`;
+
+const ListItemLink = props => <ListItem button component={RouterLink} {...props} />;
+
+export const MainMenu = () => {
+  const [open, setOpen] = useState(false);
+
+  const toggleDrawer = isOpen => event => {
+    if (event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+      return;
+    }
+
+    setOpen(isOpen);
+  };
+
+  return (
+    <>
+      <MenuSection>
+        <LightIconButton onClick={toggleDrawer(true)}>
+          <Menu />
+        </LightIconButton>
+      </MenuSection>
+
+      <Drawer anchor="left" open={open} onClose={toggleDrawer(false)}>
+        <MenuTray>
+          <IconButton color="inherit" onClick={toggleDrawer(false)}>
+            <Close />
+          </IconButton>
+        </MenuTray>
+        <StyledList onClick={toggleDrawer(false)} onKeyDown={toggleDrawer(false)}>
+          <ListItem>
+            <MenuHeading variant="h3">Menu</MenuHeading>
+          </ListItem>
+          <ListItemLink to="/">
+            <ListItemIcon>
+              <Home />
+            </ListItemIcon>
+            <ListItemText primary="Home" />
+          </ListItemLink>
+          <ListItemLink to="/about">
+            <ListItemIcon>
+              <ImportContacts />
+            </ListItemIcon>
+            <ListItemText primary="About LESMIS" />
+          </ListItemLink>
+          <ListItemLink to="/contact">
+            <ListItemIcon>
+              <ContactMail />
+            </ListItemIcon>
+            <ListItemText primary="Contact us" />
+          </ListItemLink>
+        </StyledList>
+        <StyledDivider />
+        <StyledList component="nav" aria-label="secondary mailbox folders">
+          <ListItem>
+            <TupaiaText>
+              Powered by <Link href="https://tupaia.org">Tupaia</Link>
+            </TupaiaText>
```

Comment:
Once again not at all my area, but I feel that maybe this `powered by` component is a bit more front-and-center than I'm used to seeing it? Perhaps if it will aligned to the bottom of the menu component instead of with the other list items?
Also, not sure what our policy is on this, but wondering if https://www.info.tupaia.org might be a bit of a better URL to redirect people to? tupaia.org will just hit them with a login screen immediately...

Reply:
Same as above, I'm just going to roll with the designs. In terms of the url though, I think you are right that the info website is a better choice

Reply:
Yes I agree the info website is a better choice for the url. In regards to the design, I'll once again just roll with the designs.